cd_database/migrations/1714726277027648_events.up.sql

```diff
@@ -4,7 +4,7 @@ CREATE TABLE IF NOT EXISTS events
     timestamp TIMESTAMP,
     commitHash VARCHAR(64),
     eventType VARCHAR(32),
-    json VARCHAR(255),
+    json VARCHAR(1024),
```

Comment:
Do we really know it can't ever be bigger than 1024?
I would prefer to make it unlimited:
```suggestion
    json VARCHAR,
```
```suggestion
    json VARCHAR(1024),
```